### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by daudi13**
## In this project:

-  I created an empty To Do List placeholder using `<ul>` tags
-  I created an `Index.js` file and set an array of some simple to do task.
-  I wrote a function to iterate over the task array and populate an HTML list element for each task

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
```
Comment (rloterh) [inline webpack.config.js:3]: - [ ] OPTIONAL
_Mode_ option is not set for _webpack_. Kindly set the `mode` and either assign its value to `development` or `production` within the _module.exports_

**Webpack config :**
image

Comment (rloterh) [inline webpack.config.js:3]: - [ ]  Your app is generating errors during compilation, and hence not working as intended. Kindly ensure to fix this issue:

image
image

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
+    <title>To do</title>
+</head>
+<body>
```
Comment (rloterh) [inline src/index.html:9]: - [ ] The styling of your app design does not match the provided [design template](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Kindly ensure your design follows the design template in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).

**Your design:**
image

**The original design:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi David, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 2 above)
Comment (rloterh) [reply src/index.html:9]: @daudi13 
Sure..I've messaged you on slack, kindly send the link in the thread!

Comment (rloterh) [reply src/index.html:9]: @daudi13 
As per our discussion, some commits were made to this PR after your project has been submitted. Kindly ensure that commit/ updates are done only after review is completed for every submitted project! 

Happy coding! 😃

File: webpack.config.js (hunk 1 above)
Comment (daudi13) [reply webpack.config.js:3]: fixed.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @daudi13 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Professional README.
✅ Used webpack to bundle JavaScript.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**